scripts/genproto.sh

```diff
@@ -92,23 +92,46 @@ done
 log_callout -e "\\nRunning swagger ..."
 run_go_tool github.com/hexfusion/schwag -input=Documentation/dev-guide/apispec/swagger/rpc.swagger.json
 
+
 if [ "$1" != "--skip-protodoc" ]; then
   log_callout "protodoc is auto-generating grpc API reference documentation..."
 
-  run rm -rf Documentation/dev-guide/api_reference_v3.md
+  # API reference
+  API_REFERENCE_FILE="Documentation/dev-guide/api_reference_v3.md"
+  run rm -rf ${API_REFERENCE_FILE}
   run_go_tool go.etcd.io/protodoc --directories="api/etcdserverpb=service_message,api/mvccpb=service_message,server/lease/leasepb=service_message,api/authpb=service_message" \
-    --title="etcd API Reference" \
-    --output="Documentation/dev-guide/api_reference_v3.md" \
+    --output="${API_REFERENCE_FILE}" \
     --message-only-from-this-file="api/etcdserverpb/rpc.proto" \
-    --disclaimer="This is a generated documentation. Please read the proto files for more." || exit 2
+    --disclaimer="---
+title: API reference
+---
+
+This API reference is autogenerated from the named \`.proto\` files." || exit 2
+
+  # remove the first 3 lines of the doc as an empty --title adds '### ' to the top of the file.
+  run sed -i -e 1,3d ${API_REFERENCE_FILE}
 
-  run rm -rf Documentation/dev-guide/api_concurrency_reference_v3.md
+  # API reference: concurrency
+  API_REFERENCE_CONCURRENCY_FILE="Documentation/dev-guide/api_concurrency_reference_v3.md"
+  run rm -rf ${API_REFERENCE_CONCURRENCY_FILE}
   run_go_tool go.etcd.io/protodoc --directories="server/etcdserver/api/v3lock/v3lockpb=service_message,server/etcdserver/api/v3election/v3electionpb=service_message,api/mvccpb=service_message" \
-    --title="etcd concurrency API Reference" \
-    --output="Documentation/dev-guide/api_concurrency_reference_v3.md" \
-    --disclaimer="This is a generated documentation. Please read the proto files for more." || exit 2
+    --output="${API_REFERENCE_CONCURRENCY_FILE}" \
+    --disclaimer="---
+title: \"API reference: concurrency\"
+---
+
+This API reference is autogenerated from the named \`.proto\` files." || exit 2
+
+  # remove the first 3 lines of the doc as an empty --title adds '### ' to the top of the file.
+  run sed -i -e 1,3d ${API_REFERENCE_CONCURRENCY_FILE}
 
   log_success "protodoc is finished."
+  log_success ""
```

Comment:
Maybe:

log_warning "The files has NOT been automatically published on the website."